cluster-provision/images/vm-image-builder/build-containerdisk.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -exuo pipefail
+
+SCRIPT_DIR="$(
+  cd "$(dirname "$BASH_SOURCE[0]")"
+  pwd
+)"
+
+function cleanup() {
+  echo "[build-container-disk] cleanup"
+  rm -f Dockerfile
+
+  if [ $? -ne 0 ]; then
+    rm -f "$tar_file"
+    docker image rm "$full_image_tag"
+  fi
+}
+
+image_name=$1
+tag=$2
+vm_image_file=$3
+
+export TEMPLATE_DOCKERFILE=${TEMPLATE_DOCKERFILE:-$SCRIPT_DIR/Dockerfile.containerdisk}
```

Comment:
Why do you need to export this? Why is using it later?

Reply:
You right it should not be exported, I will remove `export` in following commits